ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py

```diff
@@ -491,3 +492,15 @@ def test_paddle_nll_loss(
         ignore_index=ignore_index,
         reduction=reduction,
     )
+def softmax_cross_entropy(logits, labels):
```

Comment:
Hey! Thanks for contributing! Please link the issue you're working on to the PR, and the frontend function should be added to the functional file, this test file should only consist of the test functions, formatted like the existing ones. The outermost logits, labels and print statements should not be included as well, thanks!